legend-engine-xt-relationalStore-trino-execution-tests/src/test/java/org/finos/legend/engine/plan/execution/stores/relational/connection/test/ExternalIntegration_TestConnectionAcquisitionWithFlowProviderWithDelegatedKerberosAuth_Trino.java

```diff
@@ -0,0 +1,90 @@
+// Copyright 2021 Goldman Sachs
```

Comment:
this file should be deleted too?

Reply:
This is getting used as part of the TrinoDelegatedKerberosAuth Test